### PR TITLE
Bugfix for SQS max_fetch type

### PR DESCRIPTION
**Pull request by jrauen**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
Params are passed in as strings but the max_fetch parameter wasn't converted to int before being used.

## Minimum version of Cortex XSOAR
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/AWS-SQS/Integrations/AWS-SQS/AWS-SQS.py [hunk 1]
```diff
@@ -214,7 +214,7 @@ def main():
     timeout = params.get('timeout')
     retries = params.get('retries') or 5
     aws_queue_url = params.get('queueUrl')
-    max_fetch = min(params.get('max_fetch', 10), 100)
+    max_fetch = min(int(params.get('max_fetch', 10)), 100)
```
Comment (JasBeilin) [inline Packs/AWS-SQS/Integrations/AWS-SQS/AWS-SQS.py:217]:
```suggestion
    max_fetch = min(arg_to_number(params.get('max_fetch', 10)), 100)
```

File: Packs/AWS-SQS/ReleaseNotes/1_2_9.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#### Integrations
+##### AWS - SQS
+- Fixed an issue where the `max_fetch` parameter was not parsed into an integer.
```
Comment (JasBeilin) [inline Packs/AWS-SQS/ReleaseNotes/1_2_9.md:3]:
```suggestion
- Fixed an issue where the *max_fetch* parameter was not parsed into an integer.
```

**Review by JasBeilin — CHANGES_REQUESTED** [review]

**Comment by bakatzir** [comment]
Hi @jrauen, we could not ignore your multiple contributions to our repository
https://github.com/demisto/content/pulls?q=is%3Apr+author%3Ajrauen
And wanted to send you some Cortex/PANW swag, as a small token of appreciation.
can you share your email/DFIR(https://start.paloaltonetworks.com/join-our-slack-community) so we can coordinate this?

**Comment by jrauen** [comment]
DFIR: badarsebard

**Comment by jrauen** [comment]
> Hi @jrauen, Thank you for your contribution! Please check my suggestions. Also note that you have not updated the integration's version in the metadata file. Please fix so we can move forward :)

Apologies for the confusing previous commit, I had rebased my branch onto master out of habit and it cluttered things up. I've reverted to the original parent commit and things look clean. You can verify the changes you've requested in the latest commit.

**Review by JasBeilin — APPROVED** [review]

**Pull request merged**